pkg/inference/scheduling/loader.go

```diff
@@ -364,6 +371,8 @@ func (l *loader) load(ctx context.Context, backendName, model string, mode infer
 
 	// Loop until we can satisfy the request or an error occurs.
 	for {
+		slot := -1
```

Comment:
Curious on the need for relocating this variable - does the `goto` require it?

Reply:
Yep, the compiler does not like jumping over variable declarations.